CanalTP/NavitiaSDKUX_android
# Feature sdk 128 disruption badge

## How to test

Edit the MainActivity.java in sample app:

    intent.putExtra("initOriginId", "stop_area:QUI:SA:ARCOMQUIKVYC");
    intent.putExtra("initDestinationId", "stop_area:QUI:SA:ARCOMQUIVIOL");

See the disruption badge on line 10
